### PR TITLE
D8CORE-2490: Updated NPM Packages and styles for new alerts.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Updates to the latest Decanter alerts release.

# Review By (Date)
- Sept 3rd

# Criticality
- 1/10

# Review Tasks
- Review code
- Review aesthetics
- Try it out

Screen Shot 2020-09-01 at 10 21 10 PM

### CODE REVIEW

File: lib/scss/stanford_notifications.scss [hunk 1]
```diff
@@ -86,3 +86,11 @@ $fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
     }
   }
 }
+
+.notification-list {
+  .su-alert--success {
+    * {
+      color: $su-color-white;
+    }
```
Comment (pookmish) [inline lib/scss/stanford_notifications.scss]: Should links be a different color?

Comment (sherakama) [reply lib/scss/stanford_notifications.scss]: yeah, I suppose so.

**Comment by sherakama** [comment]
Screen Shot 2020-09-02 at 9 58 35 AM

File: lib/scss/stanford_notifications.scss [hunk 2]
```diff
@@ -88,9 +88,18 @@ $fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
...
 
 .notification-list {
-  .su-alert--success {
-    * {
-      color: $su-color-white;
+  .su-alert {
+    a {
+
+      &,
+      &:link {
```
Comment (pookmish) [inline lib/scss/stanford_notifications.scss:95]: wat dis? I haven't seen `:link` before

Comment (sherakama) [reply lib/scss/stanford_notifications.scss:95]: LMGTFY
https://developer.mozilla.org/en-US/docs/Web/CSS/:link

Comment (pookmish) [reply lib/scss/stanford_notifications.scss:95]: alrighty then

**Comment by sherakama** [comment]
Thanks.

**Pull request merged**